README.md

```diff
@@ -7,6 +7,27 @@ This is the development repository for the uDALES (urban Dutch Atmospheric Large
 <img src="docs/assets/images/fielddump_slice_3D.102.png" alt="uDALES" height="256"> 
 </div>
 
+## Overview
+
+With continuing urbanization, challenges associated with the urban environment such as air quality, heat islands, pedestrian thermal comfort, and wind loads on tall buildings, are increasingly relevant. Our ability to realistically capture processes such as the transport of heat, moisture, momentum and pollutants, and those of radiative transfer in urban environments is key to understanding and facing these challenges (Oke et al., 2017).
+
+The turbulent nature of the urban flow field and the inherent heterogeneity and wide range of scales associated with the urban environment result in a complex modelling problem. Large-eddy simulation (LES) is an approach to turbulence modelling used in computational fluid dynamics to simulate turbulent flows over a wide range of spatial and temporal scales. LES is one of the most promising tools to model the interactions typical of urban areas due to its ability to resolve the urban flow field at resolutions of _O_(1 m, 0.1 s), over spatial domains of _O_(100 m), and time periods of _O_(1 h). Although there are many scalable LES models for atmospheric flows, to our knowledge, only few are capable of explicitly representing buildings and of modelling the full range of urban processes (e.g. PALM-4U; Resler et al. 2017; Maronga et al. 2020; or OpenFoam; Weller et al. 1998)..
+
+uDALES is capable of modelling urban environments at the microclimate scale including wet thermodynamics, idealized and complex morphologies, three-dimensional urban surface energy balance and real-time pollution dispersion.
+
+## Publications and validation
+
+To-date, uDALES has been used to study a number of research questions:
+
+| Research application                                         | Reference                                                                       |
+| ------------------------------------------------------------ | ------------------------------------------------------------------------------- |
+| Urban boundary layers/ boundary-layer meteorology            | Grylls et al. (2019); Sutzl et al. (2020)                                       |
```

Comment:
Also I'd like to have my dots please :) Sützl et al. (2021)

Reply:
Of course!